float64 filtered
float64 raw